docs/timeline/index.html

```diff
@@ -1322,9 +1322,20 @@ <h2 id="Methods">Methods</h2>
       </td>
     </tr>
 
-  </table>
-
+     <tr>
+      <td>zoomIn(percentage)</td>
+      <td>none</td>
+      <td>Zoom in the current visible window. The parameter <code>percentage</code> can be a <code>Number</code>. If the parameter value of <code>percentage</code> is null, the window will be left unchanged.
```

Comment:
We should make sure what a `percentage` value is. Something between [0..1] or [0..100]?